### PR TITLE
refactor(link)!: remove text-link tokens for link t3 tokens

### PR DESCRIPTION
**Pull request by jinlee93**
[EDS-1009]

### Summary:
- removes `text-link` tokens since link now has t3 tokens

### Test Plan:

- [x] CI tests / new tests are not applicable
  - no visual regressions

[EDS-1009]: https://czi-tech.atlassian.net/browse/EDS-1009?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/components/Tabs/Tabs.module.css [hunk 1]
```diff
@@ -105,7 +105,7 @@
    * Tabs link within active tabs item
    */
   .tabs__item.eds-is-active & {
-    color: var(--eds-theme-color-text-link-brand);
+    color: var(--eds-theme-color-link-brand-text);
```
Comment (jinlee93) [inline src/components/Tabs/Tabs.module.css:108]: This needs to be synced with design for maybe creating Tabs own t3 token

**Review by booc0mtaco — APPROVED** [review]
this looks good, although am i missing something? no files in design-tokens have changed, so how were these tokens in variables-nest.json?

**Comment by jinlee93** [comment]
> this looks good, although am i missing something? no files in design-tokens have changed, so how were these tokens in variables-nest.json?

The tokens were added back in their respective json files during a rebase, re-removed them and re-pushed

**Comment by booc0mtaco** [comment]
@jinlee93 ah makes sense now

**Comment by jinlee93** [comment]
I'm gonna hold off on this one since it seems to need some sync

**Comment by booc0mtaco** [comment]
Cleaning up existing tokens that were already marked as deprecated. Should be safe to remove

**Pull request merged**